src/vendor/wordpress/styles/_core-blocks.scss

```diff
@@ -158,6 +158,17 @@ $wp-button-gap: size.$spacing-gap-button-group-default;
     max-inline-size: calc(50% - 0.5em); // 2
   }
 
+  /// By default, WordPress uses table styles to constrain captions to the size
+  /// of the image. This makes sense for left- and right-aligned image blocks,
+  /// but for consistency with our non-block `figure` elements, we allow
+  /// captions to go as wide as they need for centered elements.
+  .aligncenter {
+    &,
+    > figcaption {
+      display: block;
+    }
+  }
```

Comment:
I considered using `display: initial`, but that actually doesn't get us what we want since images aren't block display by default.

I also considered using `display: block` for `.wp-block-image .aligncenter` but `display: inherit` for the `figcaption`. But I couldn't think of a scenario where we'd actually _want_ that value to inherit... if I set a class on `.aligncenter` that changed its display to `flex`, I wouldn't want that to also impact the `figcaption`.

So I landed on just setting both to `display: block` explicitly.